### DISCUSSION TITLE
[zodios-react] path parameter for mutation

### DISCUSSION THREAD

**Discussion by QzCurious** [ecyrbe/zodios#268] [General]
sample code [here](https://codesandbox.io/s/zodios-react--immutable-path-parameter-5whkp4)

It's a bit awkward that I need to bind path parameter to a `useXXX` for a mutation.
I want to know if it's a design decision or if it can be move to `mutate()`.

![image](https://user-images.githubusercontent.com/38932402/209071751-81ce817c-5e8c-4a05-bac2-c2b10ad818f6.png)

**Comment by ecyrbe** [comment]
It's a design decision. Params are not mutation data. Only body is mutation data.
